pkgs/applications/networking/gopher/sacc/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchgit, ncurses
+, patches ? [] # allow users to easily override config.def.h
+}:
+
+stdenv.mkDerivation rec {
+  pname = "sacc";
+  version = "1.01";
+
+  src = fetchgit {
+    url = "git://bitreich.org/sacc";
+    rev = version;
+    sha256 = "0n6ghbi715m7hrxzqggx1bpqj8h7569s72b9bzk6m4gd29jaq9hz";
+  };
+
+  inherit patches;
+
+  buildInputs = [ ncurses ];
+
+  postPatch = ''
+    substituteInPlace config.mk \
+      --replace curses ncurses \
+      --replace "/usr/local" "$out"
+    '';
+
+  postInstall = ''
+    install -Dm644 LICENSE $out/share/sacc/LICENSE
```

Comment:
Is this some custom license?  Maybe it could be added to the licenses list.

Reply:
Kinda zoned out while packaging this, yeah installing the license is kind of unnecessary.

It is in fact the ISC license except they use "and" instead of "and/or" in one instance which is probably insignificant.

Reply:
Fixed.